src/index.js

```diff
@@ -5,4 +5,8 @@ const hs = require('./health-score');
 const startTime = new Date();
 hs.compile(core, github)
   .then((score) => hs.report(startTime, core, github, score))
-  .then(console.log);
+  .then(console.log)
+  .catch((err) => {
+    core.setFailed('Failed to check up on the health score!');
+    console.error(err);
+  });
```

Comment:
Added the `catch` here at the end since I think `hs.report` was still attempted if the `catch` was before it. Open to adjustments though!

<details>
<summary>With the earlier catch</summary>

```
$ node
> healthscore=require('./dist/index')
...
::error::Failed to check up on the health score!
TypeError: core.getInpu is not a function
    at Object.compileScore [as compile] (/Users/ethan.zimbelman/programming/tools/slack-health-score/src/health-score.js:15:1)
    at /Users/ethan.zimbelman/programming/tools/slack-health-score/src/index.js:6:1
    at /Users/ethan.zimbelman/programming/tools/slack-health-score/dist/index.js:71959:3
    at Object.<anonymous> (/Users/ethan.zimbelman/programming/tools/slack-health-score/dist/index.js:71962:12)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
::warning::No GitHub token found; will not report score on commit status.
0
```

</details>


<details>
<summary>With the later catch</summary>

```
$ node
> healthscore=require('./dist/index')
...
::error::Failed to check up on the health score!
TypeError: core.getInpu is not a function
    at Object.compileScore [as compile] (/Users/ethan.zimbelman/programming/tools/slack-health-score/src/health-score.js:15:1)
    at /Users/ethan.zimbelman/programming/tools/slack-health-score/src/index.js:6:1
    at /Users/ethan.zimbelman/programming/tools/slack-health-score/dist/index.js:71959:3
    at Object.<anonymous> (/Users/ethan.zimbelman/programming/tools/slack-health-score/dist/index.js:71962:12)
    at Module._compile (node:internal/modules/cjs/loader:1546:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
    at Module.load (node:internal/modules/cjs/loader:1317:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1127:12)
    at TracingChannel.traceSync (node:diagnostics_channel:315:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
```

</details>